ospool-pilot/itb/pilot/additional-htcondor-config

```diff
@@ -467,7 +467,7 @@ if [[ ! $IS_CONTAINER_PILOT || $CONTAINER_PILOT_USE_JOB_HOOK ]]; then
     # Run all jobs in singularity if we have it
     # special case - allow some OSG staff to bypass Singularity for
     # debugging and building images
-    set_condor_knob "SINGULARITY_JOB" "(HAS_SINGULARITY?:false) && (IgnoreSingularity =!= True)"
+    set_condor_knob "SINGULARITY_JOB" "(HAS_SINGULARITY?:false) && !(container_bypass == True && stringListIMember(ProjectName, \"OSG-Staff,Caltech_2024_Reitze\"))"
```

Comment:
Are you envisioning this as a first class submit command that we configure on the APs?

Reply:
No, users would have to add `+container_bypass = True` to their jobs